apps/system-e2e/src/support/commands.ts

```diff
@@ -60,18 +60,18 @@ Cypress.Commands.add('patchSameSiteCookie', (interceptUrl) => {
   }).as('sameSitePatch')
 })
 
-Cypress.Commands.add('idsLogin', ({ phoneNumber }) => {
+Cypress.Commands.add('idsLogin', ({ phoneNumber, url }) => {
```

Comment:
Maybe have some sane default?
```suggestion
Cypress.Commands.add('idsLogin', ({ phoneNumber, url='/' }) => {
```